### PR TITLE
Rc/case tile shading

### PR DESCRIPTION
**Pull request by Robert-Costello**
## Product Description

This allows for different shading of fields within a case tile.
Screenshot 2023-12-21 at 2 32 29 PM

Screenshot 2023-12-21 at 2 37 47 PM

Screenshot 2023-12-21 at 2 38 24 PM

## Technical Summary

jira ticket: https://dimagi-dev.atlassian.net/browse/USH-3952
## Feature Flag

case_list_tile
case_list_tile_custom

### Safety story

under FF
tested locally
testing on staging as well

### Rollback instructions

- [x] This PR can be reverted after deploy with no further considerations

### Labels & Review
- [x] Risk label is set correctly
- [x] The set of people pinged as reviewers is appropriate for the level of risk of the change

### CODE REVIEW

**Review by MartinRiese — APPROVED** [review]
Code looks good. I am not sure about the margin in the bottom

File: corehq/apps/cloudcare/templates/formplayer/case_list.html [hunk 1]
```diff
@@ -244,19 +244,25 @@ <h1  aria-label="<%- title %>" tabindex="0" class="page-title"><%- title %></h1>
       grid-area: <%- model.gridStyle %>;
       font-size: <%- model.fontStyle %>;
       text-align: <%- model.horizontalAlign %>;
-      <% if (!model.borderInTile) { %>
+      <% if (!model.borderInTile && !model.shadingInTile) { %>
         justify-self: <%- model.horizontalAlign %>;
         align-self: <%- model.verticalAlign %>;
```
Comment (nospame) [inline corehq/apps/cloudcare/templates/formplayer/case_list.html:249]: Are these properties just not used/usable when the tile has a border or shading applied?

**Review by nospame — APPROVED** [review]
One question but I think this looks good. It's slightly weird to me that "shading" is actually lightening, but on a default gray case tile it makes sense.

**Comment by Robert-Costello** [comment]
> I am not sure about the margin in the bottom

@MartinRiese I think a `margin-bottom` of 5px looks much better. What do you think about that?

File: corehq/apps/cloudcare/templates/formplayer/case_list.html (hunk 1 above)
Comment (Robert-Costello) [reply corehq/apps/cloudcare/templates/formplayer/case_list.html:249]: Yes, those properties are not used when the tile has either shading, border or both. If shading or border are set then `justify-self` is set to `stretch` and the padding, margin etc are set. (Personally I find the ejs syntax almost unreadable)

**Comment by Robert-Costello** [comment]
leaving linting for a separate PR. also not entirely convinced that shortening the lines of test xml makes sense here.

**Pull request merged**